packages/wasm-dpp/src/data_contract/state_transition/data_contract_create_transition/mod.rs

```diff
@@ -169,4 +172,33 @@ impl DataContractCreateTransitionWasm {
             )
             .with_js_error()
     }
+
+    // TODO: add verify_signature to other identity signed state transitions
+    #[wasm_bindgen(js_name=verifySignature)]
+    pub fn verify_signature(
```

Comment:
I'm wondering where is it called, as it's also called inside the `::validate` function